### PR TITLE
Rewrite standard/index to be a guide into the documentation

### PR DESCRIPTION
**Pull request by cartermp**
This attempts to address #1359.

### CODE REVIEW

**Comment by cartermp** [comment]
Ping @mairaw and @BillWagner for review

File: docs/standard/index.md [hunk 1]
```diff
@@ -14,65 +14,48 @@ ms.assetid: bbfe6465-329d-4982-869d-472e7ef85d93
...
 
-**Managed Memory** -- .NET automatically manages memory for you via a garbage collector. It ensures that you always reference live objects, guaranteeing that you avoid nasty problems like buffer overruns and access violations. This includes array bounds checking.
+## Porting .NET Framework Code to .NET Core
 
-**Type Safety** -- The primary .NET model for functionality and memory representation is "types". Types define shape and optionally behavior. The runtime ensures that calling code can only operate on types according to their definition and specified visibility of members, providing consistent, reliable and secure results.
+If you're looking to port an application, service, or some component of a system to .NET Core, check out [Porting to .NET Core from .NET Framework](../core/porting/index.md).
 
-Features
---------
+## Porting a NuGet package from to .NET Standard
```
Comment (BillWagner) [inline docs/standard/index.md]: From *what* to .NET Standard?

File: docs/standard/index.md [hunk 2]
```diff
@@ -14,65 +14,48 @@ ms.assetid: bbfe6465-329d-4982-869d-472e7ef85d93
...
 
-**User-defined Value Types** -- Value types are a useful category of types since they offer the semantic of "pass by value" instead of "pass by reference", as is the case for classes. Value types are most obviously usefully for numeric data. .NET enables value types for both primitive types, like integers, and user-defined types.
+If you're looking to port a NuGet package to .NET Standard, check out [Porting to .NET Core from .NET Framework](../core/porting/index.md).
```
Comment (BillWagner) [inline docs/standard/index.md]: The heading says I'm porting to .NET Standard. This text says I'm porting to .NET Core. This should be consistent.

File: docs/standard/index.md [hunk 3]
```diff
@@ -14,65 +14,48 @@ ms.assetid: bbfe6465-329d-4982-869d-472e7ef85d93
...
+* [.NET Standard Library](library.md)
+* [C# Guide](../csharp/index.md)
+* [F# Guide](../fsharp/index.md)
+* [Visual Basic Guide](../visual-basic/index.md)
+* [Native Interoperability](native-interop.md)
+* [Garbage Collection](garbagecollection/index.md)
+* [Base Types in .NET](base-types/index.md)
+* [Collections](collections/index.md)
+* [Dates, times, and time zones](datetime/index.md)
+* [Asynchronous Programming](async.md)
```
Comment (BillWagner) [inline docs/standard/index.md]: Some of these feel a bit advanced for a heading that says "fundamentals". I like that you've included them, but we might want to consider splitting into two lists. I'll suggest "fundamentals" and "major concepts", but I'm really not thrilled with that term. 

Specifically I'm thinking the interop, GC and async are more advance and more specialized.

**Review by BillWagner — COMMENTED** [review]
Looks good. Left comments for a couple fixes and one suggestion.

File: docs/standard/index.md (hunk 2 above)
Comment (cartermp) [inline docs/standard/index.md]: Yeah, right now the article is itself about porting to .NET Core, but is also relevant for .NET Standard.  This is something we'll have to clean up later, but for now I can update the text saying that the concerns are shared here.

File: docs/standard/index.md (hunk 3 above)
Comment (cartermp) [inline docs/standard/index.md]: I decided to split out the languages from here and just call them Major concepts.

File: docs/standard/index.md [hunk 4]
```diff
@@ -2,9 +2,9 @@
 title: About .NET
 description: Learn about the .NET Platform.
 keywords: .NET, .NET Core
-author: richlander
+author: cartermp
 ms.author: ronpet
-ms.date: 10/31/2016
+ms.date: 2/10/2017
```
Comment (BillWagner) [inline docs/standard/index.md]: the date should get updated.

File: docs/standard/index.md [hunk 5]
```diff
@@ -14,63 +14,51 @@ ms.assetid: bbfe6465-329d-4982-869d-472e7ef85d93
...
+The .NET Platform Guide provides a large amount of information about the .NET Platform.  Depending on your familiarity with .NET, you may wish to explore different sections of this guide and other sections of the .NET documentation.
 
-.NET is a general purpose development platform. It can be used for any kind of app type or workload where general purpose solutions are used. It has several key features that are attractive to many developers, including automatic memory management and modern programming languages, that make it easier to efficiently build high-quality applications. .NET enables a high-level programming environment with many convenience features, while providing low-level access to native memory and APIs.
+## New to .NET
 
-C#, F# and Visual Basic are popular languages that target and rely on the .NET platform. The .NET languages are known for key features such as their asynchronous programming model, language-integrated query, generic types and type system reflection. The languages also provide great options for both object-oriented and functional programming paradigms.
+If you're new to .NET, check out the [Getting Started](getting-started.md) article.
```
Comment (BillWagner) [inline docs/standard/index.md]: We've been given guidance that the gerund forms of verbs are bad for SEO. We've been updating the 'getting started' with 'get started' based on that.  /cc @mairaw

File: docs/standard/index.md [hunk 6]
```diff
@@ -14,63 +14,51 @@ ms.assetid: bbfe6465-329d-4982-869d-472e7ef85d93
...
 
-- Learn about [C#](../csharp/index.md)
-- Learn about [F#](../fsharp/index.md)
-- Browse the [.NET API Library](../../api/index.md)
-- [Introduction to the Common Language Runtime](https://github.com/dotnet/coreclr/blob/master/Documentation/botr/intro-to-clr.md)
+## New to .NET Core
 
-Fundamentals
-------------
+If you're new to .NET Core, check out [Getting Started with .NET Core](../core/getting-started.md).
```
Comment (BillWagner) [inline docs/standard/index.md]: consider 'Get started'.

File: index.md [hunk 7]
```diff
@@ -81,8 +81,8 @@ title: .NET Documentation
                                                 </div>
                                             </div>
                                             <div class="cardText">
-                                                <h3><a href="/dotnet/articles/welcome">What is .NET?</a></h3>
-                                                <p><a href="/dotnet/articles/standard/getting-started">Get Started</a></p>
+                                                <h3><a href="/dotnet/articles/welcome">.NET Platform Guide</a></h3>
+                                                <p><a href="/dotnet/articles/standard/getting-started">Getting Started</a></p>
```
Comment (BillWagner) [inline index.md]: I'd leave this as 'Get started'

File: index.md [hunk 8]
```diff
@@ -101,7 +101,7 @@ title: .NET Documentation
                                             </div>
                                             <div class="cardText">
                                                 <h3><a href="/dotnet/articles/core">.NET Core Guide</a></h3>
-                                                <p><a href="/dotnet/articles/core/getting-started">Get Started</a></p>
+                                                <p><a href="/dotnet/articles/core/getting-started">Getting Started</a></p>
```
Comment (BillWagner) [inline index.md]: I'd consider leaving this as 'get started'

File: index.md [hunk 9]
```diff
@@ -120,11 +120,11 @@ title: .NET Documentation
                                                 </div>
                                             </div>
                                             <div class="cardText">
-                                                <h3><a href="/dotnet/articles/framework/">.NET Framework Guide</a></h3>
-                                                <p><a href="/dotnet/articles/framework/get-started/">Get Started</a></p>
-                                                <p><a href="/dotnet/articles/framework/development-guide">Tutorials</a></p>
-                                                <p><a href="/dotnet/articles/framework/develop-client-apps">Client Development</a></p>
-                                                <p><a href="/aspnet/#pivot=aspnet">ASP.NET</a></p>
+                                                <h3><a href="https://msdn.microsoft.com/library/w0x726c2(v=vs.110).aspx">.NET Framework Guide</a></h3>
+                                                <p><a href="https://msdn.microsoft.com/library/hh425099(v=vs.110).aspx">Getting Started</a></p>
```
Comment (BillWagner) [inline index.md]: Consider leaving as 'get started'

File: index.md [hunk 10]
```diff
@@ -141,7 +141,7 @@ title: .NET Documentation
                                             </div>
                                             <div class="cardText">
                                                 <h3><a href="https://developer.xamarin.com/guides/cross-platform/getting_started">Xamarin</a></h3>
-                                                <p><a href="https://developer.xamarin.com/guides/cross-platform/getting_started">Get Started</a></p>
+                                                <p><a href="https://developer.xamarin.com/guides/cross-platform/getting_started">Getting Started</a></p>
```
Comment (BillWagner) [inline index.md]: 'Get started'

File: index.md [hunk 11]
```diff
@@ -161,7 +161,7 @@ title: .NET Documentation
                                             </div>
                                             <div class="cardText">
                                                 <h3><a href="/dotnet/articles/csharp">C# Guide</a></h3>
-                                                <p><a href="/dotnet/articles/csharp/getting-started">Get Started</a></p>
+                                                <p><a href="/dotnet/articles/csharp/getting-started">Getting Started</a></p>
```
Comment (BillWagner) [inline index.md]: 'Get started'

File: index.md [hunk 12]
```diff
@@ -180,7 +180,7 @@ title: .NET Documentation
                                             </div>
                                             <div class="cardText">
                                                 <h3><a href="/dotnet/articles/fsharp">F# Guide</a></h3>
-                                                <p><a href="/dotnet/articles/fsharp/tutorials/getting-started">Get Started</a></p>
+                                                <p><a href="/dotnet/articles/fsharp/tutorials/getting-started">Getting Started</a></p>
```
Comment (BillWagner) [inline index.md]: 'Get started'

File: index.md [hunk 13]
```diff
@@ -199,7 +199,7 @@ title: .NET Documentation
                                             </div>
                                             <div class="cardText">
                                                 <h3><a href="/dotnet/articles/visual-basic">Visual Basic Guide</a></h3>
-                                                <p><a href="/dotnet/articles/visual-basic/getting-started">Get Started</a></p>
+                                                <p><a href="/dotnet/articles/visual-basic/getting-started">Getting Started</a></p>
```
Comment (BillWagner) [inline index.md]: 'Get started'

**Review by BillWagner — CHANGES_REQUESTED** [review]
Overall, I really like this.

There are a few changes we should make before we merge.  See the comments on Getting started vs. Get Started.

**Comment by cartermp** [comment]
@mairaw @BillWagner Could I get another round of review?

File: docs/welcome.md [hunk 14]
```diff
@@ -13,36 +13,29 @@ ms.assetid: cb788dcf-2120-467f-9c34-c02a90e1f68f
...
-This documentation will show you how to build an app from scratch. Key sections:
+This documentation covers the breadth of .NET across platforms and languages.  You can get started with .NET and its languages in any of the following sections:
 
-- [C# Guide](csharp/index.md)
-- [F# Guide](fsharp/index.md)
-- [.NET Core API](../api/index.md)
 - [.NET Platform Guide](standard/index.md)
 - [.NET Core Guide](core/index.md)
-- [.NET Framework Guide](framework/index.md)
+- [.NET Framework on Docker Guide](framework/index.md)
```
Comment (BillWagner) [inline docs/welcome.md]: I'm confused by this change. The target link doesn't say anything about Docker.

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
@cartermp I approved all the changes requested in my earlier review. I had one question/comment left. Then, it LGTM.

File: docs/welcome.md (hunk 14 above)
Comment (cartermp) [inline docs/welcome.md]: Derp, this was before the framework docs got migrated.  I'll change this.

**Pull request merged**